pkg/service/backup/restore_worker_data.go

```diff
@@ -614,9 +614,9 @@ func (w *restoreWorker) startDownload(ctx context.Context, host, dstDir, srcDir
 }
 
 // chooseIDsForBatch returns slice of IDs of SSTables that the batch consists of.
-func (w *restoreWorker) chooseIDsForBatch(ctx context.Context, shards uint, size int) []string {
+func (w *restoreWorker) chooseIDsForBatch(ctx context.Context, size int) []string {
 	var (
-		batchSize = size * int(shards)
+		batchSize = size
```

Comment:
After removing shards count modifier, there's no need to create a new variable here.